docs/contribute/toolchain.md

```diff
@@ -41,12 +41,6 @@ based on a YAML file (`_toc.yml.in`) that contains the table of contents.
 
 [Breathe](https://www.breathe-doc.org/) is a Sphinx plugin for integrating Doxygen content.
 
-## MyST
```

Comment:
This is not the only repo that made use of MyST in the past. We should keep this for accuracy and as a reference until all repos no longer use MyST

Reply:
So far I haven't seen any repos using MyST. I suggest that since it isn't the standard, that individual components that use MyST add it to their own documentation.

Reply:
Any repo using Markdown uses MyST
By default sphinx only supports RST

Reply:
Let's use rocm-docs-core to cover this information https://github.com/ROCm/rocm-docs-core/pull/1021